python/example_code/sns/sns-python-example-list-subscriptions-by-topic.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+import boto3
+
+# Create an SNS client
+sns = boto3.client('sns')
+
+# Call SNS to list the first 100 subscriptions for the specified topic
+response = sns.list_subscriptions_by_topic(TopicArn='my-topic-arn')

```

Comment:
you should put a demo arn here.

```suggestion
    TopicArn='arn:aws:sns:region:0123456789:my-topic-arn',    
```
I'd also like to see more white space to make it easier to read the code and identify what needs to be customized. 
```suggestion
response = sns.list_subscriptions_by_topic(
    TopicArn='my-topic-arn'
)
```